src/Tribe/Models/Post_Types/Event.php

```diff
@@ -96,34 +116,18 @@ protected function build_properties( $filter ) {
 						/*
 						 * We add one second during this calculation to cope with all-day events starting on 12:00 AM.
 						 * Due to how DateTime diff works diffing two following midnights would yield a diff of 2 days.
+						 * Furthermore, a multi-day event will always last at least 2 days.
 						 */
-						$one_second = new DateInterval( 'PT1S' );
-
-						$this_week_duration = min(
+						$this_week_duration = max( $multiday, min(
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: d560de0970848aadce9f83e0e21d15a9</sub>